cypress/e2e/spotlight/spotlight.spec.ts

```diff
@@ -502,6 +502,18 @@ describe("Spotlight", () => {
                             cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "true");
                             cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
                         });
+                    cy.spotlightSearch()
+                        .type("{pageDown}")
+                        .then(() => {
+                            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "false");
+                            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
+                        });
+                    cy.spotlightSearch()
+                        .type("{pageUp}")
+                        .then(() => {
+                            cy.spotlightResults().eq(0).should("have.attr", "aria-selected", "true");
+                            cy.spotlightResults().eq(1).should("have.attr", "aria-selected", "false");
+                        });
```

Comment:
This test doesn't seem to test anything specific to the page up/down behaviour beyond it behaving identically to arrows up/down when there are only 2 results.